.github/copilot-instructions.md

```diff
@@ -0,0 +1,236 @@
+# Mines Digital Services (MDS) - GitHub Copilot Instructions
+
+**CRITICAL: Always follow these instructions first and only search for additional context if information here is incomplete or found to be in error.**
+
+## Overview
+
+MDS is a full-stack mining regulatory application for British Columbia with multiple services:
+- **Frontend**: Core Web (ministry interface), Minespace Web (proponent interface), Common (shared components)
+- **Backend**: Core API (Python Flask), Document Manager, NRIS API, Permits service
+- **Database**: PostgreSQL with Flyway migrations
+- **Infrastructure**: Docker containers orchestrated with Docker Compose and Make
+
+## Working Effectively
+
+### Prerequisites Validation
+Always run `make valid` first to check your environment. This validates:
+- Node.js v20.16.0 (exact version required)
```

Comment:
Something is wonky here! I ran `make valid` after a long time of not running it.
...did we secretly update to v20.19.0 at some point? That's what I've got locally but it doesn't match either file that references the version. Anyway, nothing seems to explode with 20.19.0. 🤷‍♀️